src/weapon/w_046.c

```diff
@@ -1,6 +1,10 @@
 // Weapon ID #46. Used by weapons:
 // Holbein dagger, Blue knuckles
 #include "weapon_private.h"
+extern u16 D_146000_8017A2B0[];
+extern u16 D_146000_8017A530[];
+extern u16 D_146000_8017A670[];
+extern u16 D_146000_8017A3F0[];
 extern u16* g_WeaponCluts[];
 extern s32 g_HandId;
 #include "shared.h"
```

Comment:
```suggestion
```
you should be able to delete these three lines

Reply:
I think they're referenced in `shared.h` — should I move the includes down below the data defines?

...but I can get rid of these lines!

```c
extern u16 D_146000_8017A2B0[];
extern u16 D_146000_8017A530[];
extern u16 D_146000_8017A670[];
extern u16 D_146000_8017A3F0[];
```

Reply:
Yeah, you would need to move `shared.h` right before `static void EntityWeaponAttack(Entity* self)` like the PR I linked above

Not sure about the `extern u16 D_146000_*`, you should be able to delete them!